msgraph/graph_service_client.py

```diff
@@ -60,3 +61,12 @@ def me(self) -> UserItemRequestBuilder:
         url_tpl_parameters["user%2Did"] = "me-token-to-replace"
 
         return UserItemRequestBuilder(self.request_adapter, url_tpl_parameters)
+
+    @property
+    def batch(self) -> 'BatchRequestBuilder':
```

Comment:
this syntax seems odd. Especially since we have the "regular syntax" for the me endpoint

Reply:
Fixed to regular syntaxt